src/ports/squids/component.ts

```diff
@@ -0,0 +1,172 @@
+import { ISquidComponent, Squid, SquidMetric } from "./types";
+import {
+  ECSClient,
+  ListServicesCommand,
+  ListTasksCommand,
+  DescribeTasksCommand,
+  ListServicesRequest,
+  UpdateServiceCommand,
+  DescribeServicesCommand,
+} from "@aws-sdk/client-ecs";
+import { IFetchComponent } from "@well-known-components/interfaces";
+import { IPgComponent } from "@well-known-components/pg-component";
+import { getPromoteQuery } from "./queries";
+import { getMetricValue, getSquidsNetworksMapping } from "./utilts";
+import { Network } from "@dcl/schemas";
+
+const AWS_REGION = "us-east-1";
+const CLUSTER_NAME = process.env.CLUSTER_NAME || "dev-main";
```

Comment:
What do you think about grabbing this environment from the config component?

Reply:
good idea, done!